packages/core/src/utilities/VoxelManager.ts

```diff
@@ -308,6 +354,78 @@ export default class VoxelManager<T> {
     voxelManager.sourceVoxelManager = sourceVoxelManager;
     return voxelManager;
   }
+
+  /**
+   * Creates a history remembering voxel manager.
+   * This will remember the original values in the voxels, and will apply the
+   * update to the underlying source voxel manager.
+   */
+  public static createLazyVoxelManager<T>(
```

Comment:
now with the documentation I see that it is not about lazy and rather history? How about we call it `createHistoryVoxelManager` then

Reply:
Fixed the docs to actually describe what is happening in it.